### PR TITLE
create en questionnaire

### PR DESCRIPTION
**Pull request by zhaofeng-shu33**
Issue #308

### CODE REVIEW

**Review by jeremy0519 — APPROVED** [review]

File: blog/en-us/need-questionnaire.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: To relieve the epedemics we need your questionnaire. Use more transparent data to improve the governance capacity of public health
```
Comment (ashfaq92) [inline blog/en-us/need-questionnaire.md:2]: title: We need you to fill given questionnaire to relieve the *epidemic.

File: blog/en-us/need-questionnaire.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+keywords: Questionnaire, governance capacity of public health
+description: To relieve the epedemics we need your questionnaire. Use more transparent data to improve the governance capacity of public health
```
Comment (ashfaq92) [inline blog/en-us/need-questionnaire.md:4]: We need you to fill given questionnaire to relieve the *epidemic.

File: blog/en-us/need-questionnaire.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+At this critical time in the fight against the coronavirus, emerging  technologies such as the Internet and big data have played an important role in the prevention and control of the epidemic. In order to make better use of the digital technology in the modernization of public health governance capabilities and the prevention and control of epedemics outbreaks, <span style="color:red">**the Data Governance Research Center of Tsinghua University has jointly designed this questionnaire with other public policy experts, social scientists, and data scientists. Carry out scientific research by combining large amounts of large data and survey data to provide useful contributions to the fight against the epidemic.**</span>。
+
+The information you provided is not only important for this project, but also contribute to the scientific decision and precise policy.
+
+This survery uses anonymous rules according to laws. We promise to keep your answers confidential under any conditions.
+
+Thanks for your participation and support!
+
+Pleasae use the following qr-code or [Click here](https://www.wjx.cn/m/55148108.aspx) to start.
```
Comment (ashfaq92) [inline blog/en-us/need-questionnaire.md:21]: *Please

File: blog/en-us/need-questionnaire.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+description: To relieve the epedemics we need your questionnaire. Use more transparent data to improve the governance capacity of public health
+author: wuhan2020 open source community
+date: 2020-02-03
+---
+
+![img](/images/blog/media/need-questionnaire-1.webp)
+
+![img](/images/blog/media/need-questionnaire-2.webp)
+
+At this critical time in the fight against the coronavirus, emerging  technologies such as the Internet and big data have played an important role in the prevention and control of the epidemic. In order to make better use of the digital technology in the modernization of public health governance capabilities and the prevention and control of epedemics outbreaks, <span style="color:red">**the Data Governance Research Center of Tsinghua University has jointly designed this questionnaire with other public policy experts, social scientists, and data scientists. Carry out scientific research by combining large amounts of large data and survey data to provide useful contributions to the fight against the epidemic.**</span>。
```
Comment (ashfaq92) [inline blog/en-us/need-questionnaire.md:13]: delete "。" in the end

**Comment by zhaofeng-shu33** [comment]
this pr has been merged.  I will use a use pr to resolve

**Pull request merged**